lib/tasks/yarn-install.js

```diff
@@ -62,7 +62,11 @@ module.exports = function yarnInstall(ui, zipFile) {
     tasks.push({
         title: 'Installing dependencies',
         task: (ctx) => {
-            const observable = yarn(['install', '--no-emoji', '--no-progress'], {
+            const yarnOpts = ['install', '--no-emoji', '--no-progress'];
+            if (ui.verbose) {
+                yarnOpts.push('verbose');
```

Comment:
wouldn't this need to be:
```suggestion
                yarnOpts.push('--verbose');
```
?

Reply:
Ha yes, updated